cluster-api/src/main/java/io/aiven/klaw/clusterapi/models/confluentcloud/Configs.java

```diff
@@ -0,0 +1,8 @@
+package io.aiven.klaw.clusterapi.models.confluentcloud;
+
+import lombok.Data;
+
```

Comment:
Can you add a comment to link to the docs or explain the diff between Config and Configs?